src/Microsoft.TemplateEngine.Cli/PostActionProcessors/ProcessStartPostActionProcessor.cs

```diff
@@ -55,5 +59,23 @@ public bool Process(IEngineEnvironmentSettings settings, IPostAction actionConfi
 
             return allSucceeded;
         }
+
+        private static bool TryGetExecutableFileFullPath(string executableFileName, string outputBasePath, out string resultExecutableFileName)
```

Comment:
NIT: `resultExecutableFileName` is not very clear name tbh as it might contain the full path.  Probably adding FullFileName is better.

Reply:
Well, to be precise, this is not a full file name in all the cases. We come up with a full path if we can find the file in the template folder, otherwise, we leave it as it is.